### PR TITLE
chore: iterate promo messages

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: packages/cli/src/Generate.ts [hunk 1]
```diff
@@ -249,9 +249,9 @@ This might lead to unexpected behavior.
 Please make sure they have the same version.`
             : ''
 
-        const tryAccelerateMessage = `Deploying your app to serverless or edge functions?
-Try Prisma Accelerate for connection pooling and caching.
-${link('https://pris.ly/cli/accelerate')}`
+        const tryAccelerateMessage = `Supercharge your Prisma Client with global database caching, scalable connection pooling and real-time database events.
+Explore Prisma Accelerate: ${link('https://pris.ly/cli/_accelerate')}
+Explore Prisma Pulse: ${link('https://pris.ly/cli/_pulse')}`
```
Comment (janpio) [inline packages/cli/src/Generate.ts:254]:
```suggestion
Explore Prisma Accelerate: ${link('https://pris.ly/cli/-accelerate')}
Explore Prisma Pulse: ${link('https://pris.ly/cli/-pulse')}`
```

**Comment by Jolg42** [comment]
@janpio Does this need to be merged for this release?

File: packages/cli/src/Init.ts [hunk 2]
```diff
@@ -362,6 +363,16 @@ export class Init implements Command {
       )
     }
 
+    const promoMessage = `Developing real-time features? Prisma Pulse lets you respond instantly to database changes.
```
Comment (janpio) [inline packages/cli/src/Init.ts:366]:
```suggestion
    const promoMessage = `Developing real-time features?
Prisma Pulse lets you respond instantly to database changes.
```

File: packages/cli/src/Generate.ts [hunk 3]
```diff
@@ -249,9 +249,9 @@ This might lead to unexpected behavior.
 Please make sure they have the same version.`
             : ''
 
-        const tryAccelerateMessage = `Deploying your app to serverless or edge functions?
-Try Prisma Accelerate for connection pooling and caching.
-${link('https://pris.ly/cli/accelerate')}`
+        const tryAccelerateMessage = `Supercharge your Prisma Client with global database caching, scalable connection pooling and real-time database events.
+Explore Prisma Accelerate: ${link('https://pris.ly/cli/-accelerate')}
+Explore Prisma Pulse: ${link('https://pris.ly/cli/-pulse')}`
```
Comment (janpio) [inline packages/cli/src/Generate.ts:254]:
```suggestion
        const tryAccelerateMessage = `Supercharge your Prisma Client with global database caching,
scalable connection pooling and real-time database events.
Explore Prisma Accelerate: ${link('https://pris.ly/cli/-accelerate')}
Explore Prisma Pulse: ${link('https://pris.ly/cli/-pulse')}`
```

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**